Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -53,11 +56,12 @@
 %% Bodies and joints.
 % -------------------
 % Create the pelvis, thigh, and shank bodies.
-pelvisMass = 30.0; pelvisHalfLength = 0.1;
+pelvisMass = 50.0 + additionalMass;
```

Comment:
I didn't check this. You and @carmichaelong can play around with mass values to make it behave like the original demo.